### ISSUE TITLE
Rewrite of "Technical overview" in progress

### ISSUE THREAD

**Issue by lars-hellstrom** [OpenMath/OpenMath.github.io#41]
Just FYI, I'm about 3/4 done with a rewrite of technical.md (though at the moment it's the last hectic days before the semester starts in Sweden, so I might get otherwise occupied). Some changes so far:
* Mention CMML as third official OM encoding.
* Mention PopCorn.
* Cut some outdated XML-hype.
* Reorder stuff a bit.

There are no commits yet though (mostly because some parts are quite unfinished).

The reason I started looking at the file in question was that it seemed the natural place to link on to a tutorial on XM-for-OM (as discussed in Edinburgh). Some of the cut XML material would be more natural there.

**Comment by kohlhase** [comment]
Lars, any progress here?

**Comment by lars-hellstrom** [comment]
> 28 sep. 2017 kl. 10:36 skrev Michael Kohlhase <notifications@github.com>:
> 
> Lars, any progress here?

Yes, what remains is pretty much to make a commit and push. I'd be the first time I do that against github, so I expect some RTFMing, but there is nothing more on my schedule today so that should be doable.

**Comment by kohlhase** [comment]
RTFM is good for you :->

**Comment by kohlhase** [comment]
Lars, if you could integrate David's changes, I will pull your PR. I would do that now, but I do not want his comments to get lost.

**Comment by davidcarlisle** [comment]
sigh sorry about the above travis commit messages. I need to add a filter in the travis to remove issue references. What happened there was I added the issue link for the OMSTD repo but travis copies the commit message when checking in a generated document to this repo so it triggered entries to the issues in this repo.
